files/es/web/css/zoom/index.md

```diff
@@ -93,7 +86,7 @@ p:hover {
 }
 ```
 
-#### Result
+#### Resultado
 
 {{EmbedLiveSample('First_example')}}
```

Comment:
Thanks for this @marcelozarate! What I had meant for the {{EmbedLiveSample}} "mistakes" is that even though they (curiously) work right now, the first argument passed to `{{EmbedLiveSample}}` is usually supposed to be a string of the heading containing all code for the live sample.

For example, this one here should (as good practice) be `{{EmbedLiveSample('Primer_ejemplo')}}` instead of the English heading. *Note that this macro has lots of unexpected behaviors, especially for `translated-content`, but I did already test this to check that the live sample would work after this change.*